ores/applications/wsgi.py

```diff
@@ -0,0 +1,56 @@
+"""
```

Comment:
Can we have separate commits and then squash when merging? It makes it easier to track what happens through the life of a pull request.